updating README for samples/tensorflow/accelerometer, and created a n… (#424)
TIBCOSoftware/flogo: samples/tensorflow/accelerometer/README.md

# Accelerometer Example for a Tensorflow Estimator

> To be able to use Machine Learning in Flogo at the moment it is necessary to use Tensorflow.  One of the easiest ways to do that is to build and train a model with a pre-built algorithm, an Estimator.  This sample builds a model to predict if someone is standin, walking, or running, with the DNNClassifier Estimator with accelerometer data. This model is then exported into "models/TB/15*" directories.  The largest number is the most recent model.

Reviewer: ```suggestion
To be able to use Machine Learning in Flogo you'll currently need to use TensorFlow.  One of the easiest ways to do that is to build and train a model with a pre-built algorithm, an Estimator.  This sample builds a model to predict if someone is standing, walking, or running, using the DNNClassifier Estimator with accelerometer data. This model is then exported into "models/TB/15*" directories.  The largest number is the most recent model.
```
